rkt/rkt.mk

```diff
@@ -32,3 +32,11 @@ include makelib/build_go_bin.mk
 $(call undefine-namespaces,LOCAL)
 # RKT_STAMP deliberately not cleared
 # RKT_BINARY deliberately not cleared
+
+manpages:
```

Comment:
/me turns a blind eye on this. The concerns I have, and which maybe I'll address later when I simplify some bits and document them, are:
- I would make it a part of `make all`.
- I would prefer the files to be generated inside `$(BUILDDIR)`
  - That would require passing that directory to the generator, so it would know where to put the files.
  - But that complicates things (make clean, deps, forwarding variables, blah blah blah), so don't bother with it for now.
- Pretty printers could be used there.
  - No big deal, since the output is rather short.
- These additions should be put above the `$(call undefine-namespaces,LOCAL)` line.